contributors/design-proposals/ha_master.md

```diff
@@ -183,10 +183,10 @@ A pseudo-code for adding a new master replica using managed DNS and a loadbalanc
   4. Update DNS to point to the load balancer.
 2. Clone existing master (create a new VM with the same configuration) including
    all env variables (certificates, IP ranges etc), with the exception of
-   `INITIAL_ETCD_CLUSTER`.
+   INITIAL_ETCD_CLUSTER.
```

Comment:
\` were needed there, it makes gfm to render them correctly like: `INITIAL_ETCD_CLUSTER`